### PR TITLE
Squashing

### LINKED ISSUE #5: Demonstrate squashing!

### PR DESCRIPTION
**Pull request by novaugust**
Closes #5
- includes some forgotten thing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,20 @@ A meetup on github workflows
...
 - woohoo
 ```
 
+- take advantage of GH features (close #1, etc)
+- clean up your history
+
+## Issues
+
+- whenever there's a problem
+- close them ~~only~~ mostly via pullrequest
```
Comment (jthopple) [inline README.md]: Typo? Shouldn't it be "pull request" not "pullrequest"?

File: README.md [hunk 2]
```diff
@@ -7,14 +7,32 @@ A meetup on github workflows
...
+
+## Scenarios
+
+### I solved a problem
+
+- great!
+- there should be an issue for it
+- squash down to one commit
+  `git rebase -i` (for interactive)
+  `git rebase -i HEAD~<# of commits`
```
Comment (jthopple) [inline README.md]: did you forget a closing `>`???

**Comment by novaugust** [comment]
@jthopple Fixed all my typos

**Comment by jthopple** [comment]
:100:

**Pull request merged**